inventory/group_vars/opentech-sjc

```diff
@@ -41,4 +41,8 @@ nodepool_zmq_publishers:
 zuul_gearman_server: zuul.internal.opentechsjc.bonnyci.org
 zuul_logs_url: http://logs.bonnyci.org
 
+zuul_components:
```

Comment:
By putting this here, any opentech-sjc host that runs the zuul role will get zuul launchers and servers, but we only want one server per deployment.